src/js/abstract/hocs/with-update.js

```diff
@@ -15,6 +17,13 @@ const withUpdate = Base =>
    * Adds attribute observation and enables **First Class Props**.
    */
   class WithUpdate extends Base {
+    static get observedAttributes() {
+      const { propTypes } = this;
+      const derivedAttributes = propTypes && Object.keys(propTypes).map(dasherize);
```

Comment:
I would have preferd that the derived attributes where transpiled during "build time" rather than during "run time". But still very good as it is

Reply:
Good point, though by runtime we generate less code 🤔

Reply:
compiled code i guess would be same but is fine as is, there are more important stuff :)